cli/cli-interpreter/src/main/java/org/infinispan/cli/interpreter/session/SessionImpl.java

```diff
@@ -117,7 +118,7 @@ public void createCache(String cacheName, String baseCacheName) {
 
          CreateCacheCommand ccc = factory.buildCreateCacheCommand(cacheName, baseCacheName);
          try {
-            rpc.invokeRemotely(null, ccc, rpc.getSyncRpcOptions());
+            rpc.blocking(rpc.invokeCommandOnAll(ccc, VoidResponseCollector.ignoreLeavers(), rpc.getSyncRpcOptions()));
```

Comment:
offtopic, this looks like a very old code. 
The method's code could be replaced with a single line: `cacheManager.administration().createCache(cacheName, baseCacheName);`

Reply:
Good point! This is the legacy code for the wfly server integration, so I'm not sure what the plan is in terms of refactoring and maintenance. @tristantarrant when are we killing these modules?